src/VaultPHP/SecretEngines/Interfaces/NamedRequestInterface.php

```diff
@@ -15,7 +15,7 @@ public function getName();
 
     /**
      * @param string $name
-     * @return void
+     * @return $this
```

Comment:
We sometimes use `@return self` and sometimes `@return $this`. Does this make any difference to the IDE or Psalm or do they both have the same effect?

Reply:
should make no diff on final classes but this is a leftover and should return this, fixed